backend/src/kitconcept/core/testing/configure.zcml

```diff
@@ -10,6 +10,7 @@
       title="kitconcept.core"
       description="Distribution used for testing."
       directory="distributions/testing"
+      profile_id="kitconcept.core:base"
```

Comment:
@ericof this is not a breaking change in plone.distribution?

Reply:
Actually, this is a fix, and it only affects `kitconcept.core` and `portalbrasil.core`, so not really a breaking change.